dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.grpcClientSecurity-1.0.feature

```diff
@@ -6,5 +6,5 @@ IBM-Provision-Capability: \
   osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.ssl-1.0))"
 IBM-Install-Policy: when-satisfied
 -bundles=io.openliberty.grpc.1.0.internal.client.security
-kind=beta
+kind=ga
```

Comment:
`WLP-Activation-Type: parallel` is needed on auto features too

Reply:
I'd added it to the monitor autofeature but missed this one!